fastlane/lib/versions.rb

```diff
@@ -25,15 +25,9 @@ def current_bundle_code
 end
 
 # Generate build number
-def generate_build_numbers
-	build = (ENV['CIRCLE_BUILD_NUM'].to_i + 3250).to_s
-
-	# android's build number goes way up because we need to exceed the old build
-	# numbers generated for the x86 build.
-	ci_build_num = build
-	build = ((2 * 1_048_576) + build.to_i).to_s if lane_context[:PLATFORM_NAME] == :android
-
-	{ci: ci_build_num, build: build}
+def build_number
+	# Should last until ~2080 for Android.
+	DateTime.now.to_time.to_i - DateTime.parse("2014-01-01").to_time.to_i
```

Comment:
We decided to simplify the build number logic, and just use Unix epoch time. Downside: we can only submit one build a second. Upside: no more strange calculations.